app/controllers/user_controller.py

```diff
@@ -1,18 +1,37 @@
-def signup():
-    ...
+from http import HTTPStatus
+
+from app.configs.database import db
+from app.models.user_model import UserModel
+from flask import jsonify
+from sqlalchemy.orm import Query
+from sqlalchemy.orm.session import Session
```

Comment:
Don't forget to run `isort .` and then `black .` in the terminal, we may run into some unnecessary conflicts if we don't standardize this.

Reply:
Done.